src/NewTwitchApi/Auth/AuthGuzzleClient.php

```diff
@@ -8,8 +8,12 @@
 
 class AuthGuzzleClient extends Client
 {
-    private const BASE_URI = 'https://id.twitch.tv/oauth2/';
+    const BASE_URI = 'https://id.twitch.tv/oauth2/';
```

Comment:
Why are you removing `private` from these constants?

Reply:
I didn't see the need to keep it private, I can make it back to private if you'd prefer it that way

Reply:
On the other hand, I prefer to keep things private until they need to not be. Then again, that's for my own client code. With library code like this, I suppose it might be better to not have some things be private so they can be extended. I'll think about this while other things are addressed.

Reply:
I'm with @echosa on this one. Until it becomes apparent that this (and other private entities) needs to be publicly accessible, I'd prefer we keep it private.

Reply:
Agreed. Please do so, @b3none. Thanks!